storage/src/index/ordered.rs

```diff
@@ -129,7 +129,7 @@ impl<T: Translator, V: Eq> Index<T, V> {
     pub fn prev_translated_key<'a>(&'a self, key: &[u8]) -> impl Iterator<Item = &'a V> + 'a {
         let k = self.translator.transform(key);
         self.map
-            .range(..k)
+            .range((Unbounded, Excluded(k)))
```

Comment:
this is the same logic as before only more explicit to make it more consistent with how the next_translated_key implementation looks.